rss/controller.py

```diff
@@ -37,18 +37,24 @@ def get_xml(archive_id: str, version: Optional[Any]) -> Tuple[str, int, dict]:
     """
     # Get the current date and time
     date_time = datetime.datetime.now(UTC)
+    days = 1
 
     # Create the correct serializer
     if version in (VER_RSS_2_0, None):
         serializer = RSS_2_0()  # type: Serializer
     elif version == VER_ATOM_1_0:
         serializer = Atom_1_0()
     else:
-        return "", status.HTTP_400_BAD_REQUEST, {}
+        msg = "Unsupported RSS version '" + str(version) + "' requested." + \
+              "Valid options are '" + VER_RSS_2_0 + "' and '" + VER_ATOM_1_0 + "'."
+        return msg, status.HTTP_400_BAD_REQUEST, {}
```

Comment:
We've started using some of Werkzeug's HTTP Exceptions for >= 400. So you could raise a [``BadRequest``](https://werkzeug.palletsprojects.com/en/0.15.x/exceptions/#werkzeug.exceptions.BadRequest) instead (passing the message as the description).

Reply:
That worked great, thanks!